tests/io/test.xpi.js

```diff
@@ -542,19 +532,17 @@ describe('Xpi.getFileAsStream()', function() {
         expect(err.message).toContain('¡hola!');
       });
   });
-
 });
 
-describe('Xpi.getFilesByExt()', function() {
-
+describe('Xpi.getFilesByExt()', function getFilesByExtCallback() {
```

Comment:
One of the lint rules prevents unnamed functions. In our tests we have a few places where we rely on `this`, and the semantics of a fat arrow breaks that code from working . We should probably look at refactoring those tests and set `const`s instead of relying on `this`.